stdlib/private/OSLog/OSLogMessage.swift

```diff
@@ -43,13 +45,12 @@ public enum Privacy {
 /// Maximum number of arguments i.e., interpolated expressions that can
 /// be used in the string interpolations passed to the log APIs.
 /// This limit is imposed by the ABI of os_log.
-public var maxOSLogArgumentCount: Int {
-  return 48
-}
+@_transparent
+public var maxOSLogArgumentCount: Int { return 48 }
```

Comment:
Can these constants be just... constants? `static let`s?

Reply:
Sadly that will make functions reading the static lets outside the constant evaluable fragment. This is because to constant evaluate a function all its logical inputs must be known to the evaluator. If the function reads a global state, the evaluator wouldn't know the value of the global state as it wouldn't be tracking it. It will only be tracking the values of the parameter. OTOH, the existing definition is a syntactic sugar for a pure, zero-argument function.